src/helpers/map.js

```diff
@@ -73,18 +73,9 @@ export default {
      * @returns {Object} the sections
      */
     getSections: function getSections(map) {
-        var parts = this.getParts(map),
-            result = {};
-
-        _.forEach(parts, function(part) {
-            var sections = part.sections;
-            if (sections) {
-                _.forEach(sections, function(section) {
-                    result[section.id] = section;
-                });
-            }
-        });
-        return result;
+        const parts = this.getParts(map) || {};
+
+        return Object.values(parts).reduce((result, part) => Object.assign(result, part.sections), {});
```

Comment:
can it happen that we don't have sections?

Reply:
Yes it can